core/src/test/scala/com/microsoft/ml/spark/Secrets.scala

```diff
@@ -48,6 +48,8 @@ object Secrets {
   lazy val AnomalyApiKey: String = getSecret("anomaly-api-key")
   lazy val AzureSearchKey: String = getSecret("azure-search-key")
   lazy val BingSearchKey: String = getSecret("bing-search-key")
+  lazy val TranslatorKey: String = getSecret("translator-key")
+  lazy val TranslatorName: String = getSecret("translator-name")
```

Comment:
What does the translator name do? is it state or a secret?

Reply:
It's the service name 
![image](https://user-images.githubusercontent.com/82044803/125048179-3c0a6400-e0d2-11eb-8fb0-f0c3641726de.png)

Reply:
Weird